templates/database_cleaner_rspec.rb

```diff
@@ -1,21 +1,28 @@
 RSpec.configure do |config|
+  config.use_transactional_fixtures = false
```

Comment:
I like having this here, but I am mildly concerned a viewer won't realize it's been moved out.